packages/terra-dialog-modal/tests/wdio/dialog-modal-spec.js

```diff
@@ -0,0 +1,47 @@
+/* global browser, Terra */
```

Comment:
Need to add `before` inside this comment so linter picks it up as a global value.